bookmarks/templates/bookmarks/opensearch.xml

```diff
@@ -0,0 +1,13 @@
+<Url type="text/html" method="GET" template="{{request.scheme}}://{{request.get_host}}/bookmarks?q={searchTerms}&amp;client=opensearch" />
+<Url type="application/x-suggestions+json" method="GET" template="{{request.scheme}}://{{request.get_host}}/api/bookmarks/?limit=15&amp;offset=0&amp;q={searchTerms}" />
+<Query role="example" searchTerms="Linkding" />
+<moz:SearchForm>{{request.get_host}}</moz:SearchForm>
```

Comment:
The absolute URL can be generated by Django, see the settings view for reference, which generates an absolute URL for the bookmark form:
https://github.com/sissbruecker/linkding/blob/4cb39fae99af0769f7a2bc8ece181085d8259b96/bookmarks/views/settings.py#L78

Preferrably this would happen in the view logic, not in the template itself.